diff -up setuptools-0.7.4/setup.py.orig setuptools-0.7.4/setup.py
--- setuptools-0.7.4/setup.py.orig	2013-06-19 15:25:24.000000000 +0200
+++ setuptools-0.7.4/setup.py	2015-02-02 20:28:53.908312140 +0100
@@ -6,7 +6,7 @@
 import re
 
 # Allow to run setup.py from another directory.
-os.chdir(os.path.dirname(os.path.abspath(__file__)))
+os.chdir(os.path.dirname(os.path.abspath(sys.argv[0])))
 
 src_root = None
 if sys.version_info >= (3,):
